### PR TITLE
fix/keadm: fix the logic of auto install script for keadm

### PR DESCRIPTION
**Pull request by ttlv**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
complete keadm
**Which issue(s) this PR fixes**:
https://github.com/kubeedge/kubeedge/issues/2057
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by GsssC** [comment]
Hello, :-) plz complete the describtion of this pr

File: keadm/cmd/keadm/app/cmd/util/common.go [hunk 1]
```diff
@@ -356,48 +360,22 @@ func installKubeEdge(componentType types.ComponentType, arch string, version str
...
-	} else {
-		try := 0
-		for ; try < downloadRetryTimes; try++ {
-			//Download the tar from repo
-			dwnldURL := fmt.Sprintf("cd %s && wget -k --no-check-certificate --progress=bar:force %s/v%s/%s",
-				KubeEdgePath, KubeEdgeDownloadURL, version, filename)
-			if _, err := runCommandWithShell(dwnldURL); err != nil {
+		fmt.Println("Expected or Default KubeEdge version", version, "is already downloaded and will checksum for it.")
+		if success, _ := checkSum(filename, checksumFilename, version); !success {
+			cmdStr := fmt.Sprintf("cd %s && rm -f %s", KubeEdgePath, filename)
```
Comment (GsssC) [inline keadm/cmd/keadm/app/cmd/util/common.go:365]: If fail to checkSum, I think it is better to get a user permission `y or yes`  then `remove file and reDownload`.
if no, log a warning and still use unSafe binaray to install kubeedge.

Comment (ttlv) [reply keadm/cmd/keadm/app/cmd/util/common.go]: I don't think user would like to continue to install  when uses konws the file is broken, but it's just a personal opinion, I have added selectone option for it  :-)

Comment (ttlv) [reply keadm/cmd/keadm/app/cmd/util/common.go]: ok, It seems that I have misunderstood your description,I have changed the logic of the code and if you have any ideas about the code.Welcome to have a talk with me .thank you :-)

**Pull request closed without merging**